modules/go/Makefile.build

```diff
@@ -18,13 +18,21 @@ else
 endif
 
 ## Install dependencies
-go/deps: $(GLIDE)
+go/deps: $(GO)
+	$(call assert-set,GO)
+	$(GO) get
+	$(GO) mod tidy
+
+## DEPRECATED Install dependencies
+go/glide-deps: $(GLIDE)
+	@echo "DEPRECATED: Please update to 'go mod' and use go/deps"
 	$(call assert-set,GLIDE)
 	$(GLIDE) install --strip-vendor
 	$(GLIDE) update  --strip-vendor
```

Comment:
```suggestion
	exit 1
```

Reply:
Maybe move to `Makefile.deprecated`

Reply:
Let's just remove it altogether in that case